examples/fortran/SHMag/meson.build

```diff
@@ -0,0 +1,11 @@
+TestSHMag = executable(
+  'SHMag',
+  'SHMag.f95',
+  link_with: shtools)
+
+test(
+  'SHMag',
+  TestSHMag,
+  args: [
+    meson.current_source_dir() / '../../ExampleDataFiles',
+  ])
```

Comment:
This test timed out on my laptop. I would suggest adding this argument to `test`:
```
timeout: 60
```